src/main/resources/templates/smallfull/criteria.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+    <head>
+      <title>Full Accounts Criteria</title>
+    </head>
+    <body>
+    <div id="criteria-main-content" layout:fragment="content">
+        <form id="criteria-form" th:action="@{''}" th:object="${criteria}" class="form" method="post">
+
+            <div th:replace="fragments/globalErrors :: globalErrors"></div>
+
+            <h1 id="page-title" class="heading-xlarge">
+                File full accounts
+            </h1>
+
+            <h2 class="heading-medium" id="criteria-sub-heading">Criteria</h2>
+
+            <p id="criteria-overview">Check your prepared accounts to see if you meet the criteria.</p>
+
+            <p id="criteria-list-heading">You can file full accounts here if:</p>
+
+            <ul class="list list-bullet text" id="criteria-list">
+                <li>you don't need to include a directors' report or profit and loss account</li>
+                <li>you don't need to change the date the accounts are made up to</li>
+                <li>your balance sheet doesn’t have 'fixed asset investments', 'current asset investments',
+                    'revaluation reserve', or 'intangible assets'</li>
+                <li>you only need to add notes (if applicable) for 'accounting policies', 'tangible assets',
+                    'stocks', 'debtors', 'creditors - amounts falling due within one year',
+                    or 'creditors - amounts falling due after one year'</li>
+            </ul>
+
+            <p id="criteria-additional-criteria-list-heading">Your full accounts must meet at least 2 of the following:</p>
+
+            <ul class="list list-bullet text" id="criteria-additional-criteria-list">
+                <li>turnover is not over £10.2 million</li>
+                <li>balance sheet total is not over £5.1 million</li>
+                <li>average number of employees is not over 50</li>
+            </ul>
+
+            <h2 class="heading-medium" id="citeria-submit-heading">Do your prepared full accounts meet the criteria?</h2>
+
+            <fieldset class="form-group" th:classappend="${#fields.hasErrors('isCriteriaMet')} ? 'error' : ''">
+                <div class="form-group">
```

Comment:
We might need here the <legend> element, I remembered James Francis to mention something about it as it is used by some screen readers.

Reply:
Which element?

Reply:
`<legend>`, I've just checked the prototype and I noticed that this element was there.

Reply:
Added, thanks